utils/python/CIME/compare_namelists.py

```diff
@@ -71,11 +71,15 @@ def _interpret_value(value_str, filename):
         new_tokens = []
         for token in tokens:
             if "*" in token:
-                # the following ensure that the following to namelist settings trigger a match
-                # nmlvalue = 1,1,1 versus nmlvalue = 3*1
-                sub_tokens = [item.strip() for item in token.split("*")]
-                expect(len(sub_tokens) == 2, "Incorrect usage of multiplication in token '%s'" % token)
-                new_tokens.extend([sub_tokens[1]] * int(sub_tokens[0]))
+                try:
+                    # the following ensure that the following to namelist settings trigger a match
+                    # nmlvalue = 1,1,1 versus nmlvalue = 3*1
+                    sub_tokens = [item.strip() for item in token.split("*")]
+                    expect(len(sub_tokens) == 2, "Incorrect usage of multiplication in token '%s'" % token)
+                    new_tokens.extend([sub_tokens[1]] * int(sub_tokens[0]))
+                except:
+                    # User probably did not intend to use the * operator as a namelist multiplier
+                    new_tokens.append(token)
```

Comment:
I want to make sure you are getting the right answers for the right reason.
For example, in the fire_emis example below, for the token, `'pom_a1 = 1.4*OC'`, you end up with
`sub_tokens[0] == 'pom_a1 = 1.4'` and `int(sub_tokens[0])` throws an error.
Is that what you intended? If so, please add a comment before that line so the intent is clear.